Comparator.lib

```diff
@@ -300,6 +299,33 @@ X V+ 5 -100 300 150 D 50 50 1 1 I
 ENDDRAW
 ENDDEF
 #
+# LMV393
+#
+DEF LMV393 U 0 5 Y Y 3 L N
+F0 "U" 150 150 50 H V C CNN
+F1 "LMV393" 250 -150 50 H V C CNN
+F2 "" 0 0 50 H I C CNN
+F3 "" 0 0 50 H I C CNN
+$FPLIST
+ SOIC*3.9x4.9mm*P1.27mm*
+ TSSOP*4.4x3mm*P0.65mm*
+ VSSOP*3.0x3.0mm*P0.65mm*
+ VSSOP*2.3x2mm*P0.5mm*
```

Comment:
I only see the SOIC8 package in the dataseet.

Reply:
The first page is incomplete.
See ordering information on page 14f.

Reply:
For reference
http://www.ti.com/packaging/docs/searchproductbypackage.tsp

DDU -> VSSOP 2.3x2mm P0.5
DGK -> VSSOP 3x3mm P0.65
D -> SOIC 3.9x4.9mm P1.27
PW -> TSSOP 4.4x4mm P0.65